daemons/fenced/fenced_history.c

```diff
@@ -288,9 +288,26 @@ stonith_local_history_diff(GHashTable *remote_history,
             while (g_hash_table_iter_next(&iter, NULL, (void **)&op)) {
                 xmlNode *entry = NULL;
 
-                if (remote_history &&
-                    g_hash_table_lookup(remote_history, op->id)) {
-                    continue; /* skip entries broadcasted already */
+                if (remote_history) {
+                    remote_fencing_op_t *remote_op =
+                        g_hash_table_lookup(remote_history, op->id);
+
+                    if (remote_op) {
+                        if (remote_op->state != st_failed
+                            && remote_op->state != st_done
+                            && (op->state == st_failed
+                                || op->state == st_done)) {
+
+                            crm_debug("Broadcasting operation %.8s (state=%s) to "
+                                       "update the outdated pending one "
```

Comment:
not sure what our current guidelines say but the rest in the code in this file aligns independently of the tab-space in these cases.